File: DataWarehouse/packages/client/modules/common/classes/Venue.ts

export class Venue {

    public name: string;
    public city: string;
    public state: string;
    public comment: string;

    public constructor( name_: string, city_: string, state_: string, comment_: string ) {
        this.name = name_;
        this.city = city_;
        this.state = state_;
        this.comment = comment_;
    }
}

File: GeneralServices/GeneralServices/Classes/Concert/Venue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GeneralServices.Classes
{
    public class Venue
    {
        public int venueID { get; set; }
        public string venueName { get; set; }
        public string city { get; set; }
        public string state { get; set; }
        public string comment { get; set; }
        public AuditInfo auditInfo { get; set; }

        #region Constructors

        /// <summary>
        /// Default Constructor
        /// </summary>
        public Venue()
        {

        }

        #endregion Constructors
    }
}
